src/hotspot/share/opto/phasetype.hpp

```diff
@@ -71,6 +71,12 @@
   flags(AFTER_RANGE_CHECK_ELIMINATION,  "After Range Check Elimination") \
   flags(BEFORE_PRE_MAIN_POST,           "Before Pre/Main/Post Loops") \
   flags(AFTER_PRE_MAIN_POST,            "After Pre/Main/Post Loops") \
+  flags(BEFORE_POST_LOOP,               "Before Post Loops") \
+  flags(AFTER_POST_LOOP,                "After Post Loops") \
+  flags(BEFORE_REMOVE_EMPTY_LOOP,       "Before Remove Empty Loops") \
+  flags(AFTER_REMOVE_EMPTY_LOOP,        "After Remove Empty Loops") \
+  flags(BEFORE_ONE_ITERATION_LOOP,      "Before Replacing One Iteration Loops") \
+  flags(AFTER_ONE_INTERATION_LOOP,      "After Replacing One Iteration Loops") \
```

Comment:
There is a typo for `AFTER_ONE_INTERATION_LOOP` -> `ITERATION`

Nit: We only apply it for one loop and thus you can remove the trailing `s`.
```suggestion
  flags(BEFORE_POST_LOOP,               "Before Post Loop") \
  flags(AFTER_POST_LOOP,                "After Post Loop") \
  flags(BEFORE_REMOVE_EMPTY_LOOP,       "Before Remove Empty Loop") \
  flags(AFTER_REMOVE_EMPTY_LOOP,        "After Remove Empty Loop") \
  flags(BEFORE_ONE_ITERATION_LOOP,      "Before Replacing One Iteration Loop") \
  flags(AFTER_ONE_ITERATION_LOOP,      "After Replacing One Iteration Loop") \
```